samples/hello-coroutines/build.gradle.kts

```
@@ -2,7 +2,7 @@ import org.jetbrains.kotlin.gradle.dsl.Coroutines
 
 plugins {
     application
-    kotlin("jvm") version "1.3.0"
+    kotlin("jvm") version "1.3.10"
```

Consider this instead?

```suggestion
    kotlin("jvm") version KotlinVersion.CURRENT.toString()
```

---

Thanks for the suggestion. We want the samples to advertise the version explicitly.